### PR TITLE
Us18/update item status

### PR DESCRIPTION
**Pull request by thawley2**
Added form to the merchant invoice show page to change the status of each invoice_item to either pending/packaged/shipped.  
Created new Merchant::InvoiceItems_controller with an update action. 
Added My Invoice and My Items links to the header partial and included the partial on all merchant pages. (Still need to write tests for these).

### CODE REVIEW

File: app/controllers/merchants/invoice_items_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Merchants::InvoiceItemsController < ApplicationController
```
Comment (sicknacco) [inline app/controllers/merchants/invoice_items_controller.rb:1]: So cool this worked out!

File: app/models/merchant.rb [hunk 2]
```diff
@@ -55,9 +55,9 @@ def uniq_invoices
   end
 
   def invoice_items_data(invoice_id)
-    items.joins(:invoices)
-    .select("items.*, invoice_items.quantity, invoice_items.unit_price as sold_price, invoice_items.status as invitm_status")
-    .where(invoices: {id: invoice_id})
+    invoice_items
```
Comment (sicknacco) [inline app/models/merchant.rb:58]: So much cleaner!

File: app/views/merchants/_header.html.erb [hunk 3]
```diff
@@ -1,3 +1,7 @@
 <h1>Little Esty Shop</h1>
 
-<h2><%= merchant %></h2>
\ No newline at end of file
+<h2><%= merchant %></h2>
+
+<%= link_to "Dashboard", merchant_dashboard_path(merchant_id) %>
```
Comment (sicknacco) [inline app/views/merchants/_header.html.erb:5]: This was a good call to add to the partial so it updated on all pages

**Review by sicknacco — APPROVED** [review]
Solid teamwork!

**Review by abwmn — APPROVED** [review]
congrats guys on finishing all the user stories!!

**Pull request merged**